web/src/actions/apd.test.js

```diff
@@ -167,7 +167,7 @@ describe('apd actions', () => {
       const store = mockStore();
       fetchMock.onDelete('/apds/apd-id').reply(400);
 
-      store.dispatch(actions.deleteApd('apd-id')).then(() => {
+      return store.dispatch(actions.deleteApd('apd-id')).then(() => {
```

Comment:
These tests were buggy because if the promise rejected - which it would sometimes due - Jest didn't know about it because the promise hadn't been returned.  So you'd see log output that it failed but it'd still exit successfully.  By returning the promise, we're letting Jest track its outcome.  Whoops on my part.  😬